include/constants.h

```diff
@@ -261,9 +278,9 @@ enum dcontrols
          * - mod_pk = kappa_eps = delta_e = 0: circular geometry [x=r.cos(theta), y=r.sin(theta)]
          * - else: stretched circular geometry
          *   [x=(1-kappa_eps)r.cos(theta)-delta_e.r^2, y=(1+kappa_eps)r.sin(theta)] with
-         *     - kappa_eps: Elongation
+         *     - kappa_eps: Elongation (Parameter kappa for the Shafranov geometry, parameters epsilon for the Czarny geometry, is set automatically depending on the geometry)
          *     - delta_e: Shafranov shift (outward radial
-         *       displacement of the centre of flux)
+         *       displacement of the centre of flux) (Parameter delta for the Shafranov geometry, parameters e for the Czarny geometry, is set automatically depending on the geometry)
```

Comment:
```suggestion
         *       displacement of the centre of flux) (For the Shafranov geometry, this parameter is denoted delta, for the Czarny geometry, it is e.)
```


I propose to recopy parts of my mod_pk suggestions above to the brief description above